### ISSUE TITLE
Aarch64 rpm is not correct?

### ISSUE THREAD

**Issue by Tokariew** [kopia/kopia#2978]
I imported key like in this instruction https://kopia.io/docs/installation/#linux-installation-using-rpm-redhat-centos-fedora
I compared that the key is identical to my x86_64 machine. But when I try to install aarch64 version of kopia on fedora i get following error:

```
Problem opening package kopia-0.12.1.aarch64.rpm
The downloaded packages were saved in cache until the next successful transaction.
You can remove cached packages by executing 'dnf clean packages'.
Error: GPG check FAILED
```
When i check how the package is signed i get following error:
```
error: kopia-0.12.1.aarch64.rpm: Header V4 RSA/SHA512 Signature, key ID fd56a556: BAD
error: kopia-0.12.1.aarch64.rpm: not an rpm package (or package manifest)
```

**Comment by 0x3333** [comment]
Same for Debian APT

```
Get:3 https://packages.kopia.io/apt stable InRelease [5,015 B]
Err:3 https://packages.kopia.io/apt stable InRelease
  The following signatures were invalid: EXPKEYSIG 2273699AFD56A556 Kopia Builder <builder@kopia.io>
```

**Comment by julio-lopez** [comment]
Is this a duplicate of #2947 ?

**Comment by Tokariew** [comment]
It was completly new instalation on fedora arm, and i imported key, following instruction here: https://kopia.io/docs/installation/#linux-installation-using-rpm-redhat-centos-fedora

**Comment by OperKH** [comment]
Looks like it was rmp issue and it already fixed in 4.18.1 - try to update - Fedora released rpm 4.18.1 today.
